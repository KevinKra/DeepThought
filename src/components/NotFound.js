import React from "react";

const NotFound = () => {
  return (
    <div>
      <h1>Not Found -- Under construction</h1>
    </div>
  );
};

export default NotFound;
