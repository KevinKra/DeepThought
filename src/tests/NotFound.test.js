import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import NotFound from "../components/NotFound";

describe("<NotFound />", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<NotFound />);
  });
  it("should match the snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
