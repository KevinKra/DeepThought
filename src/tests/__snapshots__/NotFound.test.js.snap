// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should match the snapshot 1`] = `
<div>
  <h1>
    Not Found -- Under construction
  </h1>
</div>
`;
